src/pytorch_lightning/loggers/neptune.py

```diff
@@ -272,14 +230,13 @@ def __init__(
         agg_default_func: Optional[Callable[[Sequence[float]], float]] = None,
         **neptune_run_kwargs: Any,
     ):
-        # verify if user passed proper init arguments
-        self._verify_input_arguments(api_key, project, name, run, neptune_run_kwargs)
```

Comment:
We still should verify input arguments.
This function is responsible for checking inter-parameters compatibility, not only detecting deprecated ones.

Reply:
It's still done, two lines below